### PR TITLE
OpenShift instructions

### PR DESCRIPTION
**Pull request by corinnekrych**
Given that some ios cookbook recipe used this [app deployed on OpenShift](https://github.com/aerogear/aerogear-ios-cookbook/blob/master/Authentication/Authentication/Network.swift#L25). 
Having the deployment instructions is handy.

### CODE REVIEW

File: Authentication/README.md [hunk 1]
```diff
@@ -34,4 +34,12 @@ curl -v \
...
\ No newline at end of file
+     mvn wildfly:deploy
+
+### Deploy on OpenShift
+
+To follow the instructions below, you need [rhc](https://developers.openshift.com/en/managing-client-tools.html) installed.
+
+     rhc app create bacon jboss-wildfly-8
+
+The ```bacon``` demo app is then available in [OpenShift online](http://bacon-corinnekrych.rhcloud.com/rest/grocery/bacons).
```
Comment (abstractj) [inline Authentication/README.md]: @corinnekrych I think we should move the bacon app to the aerogear account. Lemme know what do you think.

**Comment by abstractj** [comment]
@corinnekrych I'm very keen to migrate this app to the aerogear account. Although I don't know where this app is hosted. Do you have the sources? Would make sense to deprecate this repo or move to KC?

**Comment by corinnekrych** [comment]
@abstractj this is the app itself with the src:
https://github.com/aerogear/aerogear-backend-cookbook/tree/master/Authentication

**Comment by abstractj** [comment]
@corinnekrych 

Landed!

**Pull request closed without merging**